src/CoreBundle/private/modules/MTHeader/views/standard.view.php

```diff
@@ -253,13 +251,13 @@ class="nav-link dropdown-toggle <?= $userButtonStyle ?>"
                             </span>
                         </a>
                         <div class="dropdown-menu dropdown-menu-right">
-                            <a class="dropdown-item" href="<?= PATH_CMS_CONTROLLER ?>?pagedef=tableeditor&tableid=<?= $data['iTableIDCMSUser'] ?>&id=<?= $data['oUser']->id ?>&<?= urlencode('module_fnc[contentmodule]') ?>">
+                            <a class="dropdown-item" href="<?= PATH_CMS_CONTROLLER; ?>?pagedef=tableeditor&tableid=<?= $data['iTableIDCMSUser']; ?>&id=<?= $data['oUser']->id; ?>&<?= urlencode('module_fnc[contentmodule]'); ?>">
```

Comment:
Those semicolons aren't required and are omitted by most projects, and I would omit them, too (in fact I removed some in the last few days).

Reply:
CS-Fixer did this. Without changing the code style config i don`t get rid of them.

Reply:
Looks like I did it wrong :-) - OK, I don't like the semicolons in this case, but we stick with the CS-Fixer defaults.